compiler/qsc_parse/src/item.rs

```diff
@@ -33,7 +33,7 @@ pub enum Fragment {
 }
 
 pub(super) fn parse(s: &mut Scanner) -> Result<Box<Item>> {
-    let lo = s.peek().span.lo;
+    let t = s.peek();
```

Comment:
I think the fact that you're using `t` to create the error span means that you'll only report the error for the first token, not necessarily the whole doc comment.

Try using `s.span(lo)` below instead, to create a span that starts from `lo` (where we started parsing) to the current offset (at the time you create the error node).

Try adding a test case for this and see what the error span covers... It should cover the whole doc comment (both lines) not just the first token. Because the whole thing would be treated as a doc comment, were an item to follow it.

```qsharp
namespace Foo {
  /// doc comment token
  /// another doc comment token
}
```